Makefile

```diff
@@ -3,7 +3,7 @@
 PACKAGES = $(shell go list ./... | grep -v /vendor/)
 
 ifneq ($(shell uname), Darwin)
-	EXTLDFLAGS = -extldflags "-static" $(null)
+	EXTLDFLAGS = -extldflags "-static-libgcc -static-libstdc++" $(null)
 else
```

Comment:
lets add another `if` block that only uses these new settings for arm. I don't want to change the amd64 compiler flags since they have been working quite well and I don't want to introduce potential instability to the project.